app/assets/javascripts/app-level.js

```diff
@@ -19,9 +19,7 @@ $(document).ready(function() {
 	});
 
 	/* Activating Tooltips */
-	$(function () {
-  		$('[data-toggle="tooltip"]').tooltip()
-	})
+	$('.tooltipped').tooltip({delay: 50});
```

Comment:
The problem I was talking about was this (the blank black tooltip):
![assessment_show_empty_tooltip](https://cloud.githubusercontent.com/assets/3676913/21470034/0cfed830-ca34-11e6-89e7-f566f55dfe83.png)
There already seems to be a tooltip so we could just get rid of this thing entirely.